<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>op_register</class>
 <widget class="QMainWindow" name="op_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> 超级管理员界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>仓库 </string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>仓库信息管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stor_dele">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stor_add">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_num">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库位置：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="stor_code">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="locate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stor_name">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>操作员</string>
     </attribute>
     <widget class="QPushButton" name="op_dele">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>300</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QComboBox" name="stor_code_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_passwd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="op_add">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>仓库编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="op_code">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>输入密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>操作编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>操作员信息管理</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="con_passwd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>退出</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
